Integrations/integration-symantec-DLP-incident-reportintg-update.yml

```diff
@@ -0,0 +1,209 @@
+
+    class SymantecAuth(AuthBase):
+        def __init__(self, USERNAME, PASSWORD, host):
+            self.basic = HTTPBasicAuth(USERNAME, PASSWORD)
+            self.host = host
+
+        def __call__(self, r):
+            if r.url.startswith(self.host):
+                return self.basic(r)
+            else:
+                return r
+
+    SERVER_IP = demisto.params()['server-ip']
+    USERNAME = demisto.params()['credentials']['identifier']
+    PASSWORD = demisto.params()['credentials']['password']
+    SAVED_REPORT_ID = demisto.params()['saved-Report-id']
+    FETCH_TIME = demisto.params()['fetch_time']
+
+
+    wsdl = 'https://{}/ProtectManager/services/v2011/incidents?wsdl'.format(SERVER_IP)
+
+    session = Session()
+    session.auth = SymantecAuth(USERNAME,PASSWORD, "https://{}".format(SERVER_IP))
+    session.verify = False
+    transport = Transport(session=session)
+
+    client = Client(wsdl=wsdl, transport=transport)
+
+    def  get_incident_id(dtobj):
+
+        if type(dtobj) is datetime.datetime:
+            return client.service.incidentList(savedReportId=SAVED_REPORT_ID, incidentCreationDateLaterThan=dtobj)
+        else:
+            return_error('excpected datetime object')
+
+
+
+    def incidentDetail(_id):
+
+        return client.service.incidentDetail(incidentId=_id,includeViolations=True,includeHistory=True)
+        
+    
+        
+    def parse_date_range(daterange):
+        dateobj = datetime.now()
+        try:
+            number,time_unit = daterange.split(" ")
+            if time_unit.startswith("minute"):
+              start_time = end_time - timedelta(minutes=int(number))
+            if time_unit.startswith("hour"):
+                dateobj = dateobj - timedelta(hours=int(number))
+            elif time_unit.startswith("day"):
+                dateobj = dateobj - timedelta(days=int(number))
+            elif time_unit.startswith("month"):
+                dateobj = dateobj - timedelta(days=int(number)*30)
+            elif time_unit.startswith("year"):
+                dateobj = dateobj - timedelta(days=int(number)*365)
+        except Exception:
+            return_error('invalid dateformat supplied')
+        return dateobj
+
+
+
+    def fetchincidents():
+        last_run=demisto.getLastRun()
+        if 'lt_time' not in last_run:
+            last_run['lt_time'] = parse_date_range(FETCH_TIME)
+        else:
+            last_run['lt_time']= datetime.datetime.strptime(last_run['lt_time'],'%d/%m/%Y %H:%M:%S.%f')
+
+        incidentids=get_incident_id(last_run['lt_time'])
+        incidents=[]
+        if len(incidentids['incidentId']) > 0:
+            for incident1 in incidentids['incidentId']:
+                inc=incidentDetail(str(incident1))
+                incidents.append({
+                    'name':"DLP Incident #"+str(incident1),
+                    'details':  inc[0]['incident']['violatedPolicyRule'][0]['ruleName']+ "RULE has been violated"
```

Comment:
you should add here another property called `"rawJSON": json.dumps(inc[0]['incident])`
You want to allow users to map fields of incidents from DLP to incident in Demisto (that is why we need the rawJSON).
rawJSON should contain JSON string of original incident/case/alert/event from the integration

Reply:
added the rawJSON, also added the severity field